### DISCUSSION TITLE
svg to png yields a white background

### DISCUSSION THREAD

**Discussion by stanley-zenfolio** [dlemstra/Magick.NET#939] [Help] [answered]
Converting the following svg to png gives a white box. How can I get rid of the white box and make it transparent or no color?

![image](https://user-images.githubusercontent.com/50000923/119446959-0ce79100-bce4-11eb-9b48-e5148cc8f99c.png)

I used the following code to convert.
            byte[] bytes = Encoding.UTF8.GetBytes(svg);
            image = new MagickImage(bytes);

![image](https://user-images.githubusercontent.com/50000923/119446705-ba0dd980-bce3-11eb-84dc-8dea4bd97ea6.png)

**Comment by dlemstra** [comment, marked as answer]
You can change the background color of the `svg` file with the `ReadSettings`:

```C#
var settings = new MagickReadSettings()
{
    BackgroundColor = MagickColors.Transparent
};
using var image = new MagickImage(bytes, settings);
```

**Comment by stanley-zenfolio** [comment]
thanks
